gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -39,6 +40,73 @@ class FwGenericConfigDriver(object):
     def __init__(self):
         self.timeout = cfg.CONF.rest_timeout
 
+    def _configure_log_forwarding(self, url, mgmt_ip, port):
```

Comment:
We need to think about having this feature only in commercial version. Lets discuss.